### DISCUSSION TITLE
Group speakers/airplay devices

### DISCUSSION THREAD

**Discussion by marineson09** [philippe44/AirConnect#495] [General]
Is grouping supported? I have a Sonos 1 group and a Sonos 2 group I'd like to cast to with this app, does this app allow me to cast to both at the same time/create a group to cast to both at the same time by chance?

Great app by the way!

**Comment by philippe44** [comment]
If you create a CC group, it will appear as another player and when you play to it, it will be sync'd

**Reply by marineson09** [reply]
Nothing for airupnp though?

**Comment by philippe44** [comment]
ah, only Sonos
